app/controllers/application_controller.rb

```diff
@@ -2,4 +2,50 @@ class ApplicationController < ActionController::Base
   # Prevent CSRF attacks by raising an exception.
   # For APIs, you may want to use :null_session instead.
   protect_from_forgery with: :exception
+  helper_method :current_user, :current_order, :current_user_is_owner, :require_login
```

Comment:
Heck yeah! Helper methods!
One thing to consider though is whether a method needs to be in both the controller and the view.
I'm still reviewing but it seems like `:require_login` probably only happens in controllers and so doesn't need a helper method.

Posts that helped me understand this better to ensure that I didn't give you false information
http://codefol.io/posts/Where-Do-I-Put-My-Code
http://6ftdan.com/allyourdev/2015/01/28/rails-helper-methods/